frontend/src/redux/rootReducer.js

```diff
@@ -1,8 +1,10 @@
 import {combineReducers} from 'redux';
 import museumArtworks from './museumArtworkReducer';
 import userArtworks from './userArtworkReducer';
+import {firebaseReducer} from 'react-redux-firebase';
 
 export default combineReducers({
   museumArtworks,
   userArtworks,
+  firebase: firebaseReducer,
```

Comment:
I'm confused because there is a trailing comma, and when I run eslint locally it's fine.

Reply:
That's so weird.  It also is displaying errors on UserUploadForm.js but I saw that you pushed a commit for eslint fixes so maybe git is outdated for some reason?
![image](https://user-images.githubusercontent.com/42744910/88440465-9cc26300-cdd3-11ea-8543-af14aec2f3d6.png)